src/server/route/routes/project.ts

```diff
@@ -61,10 +61,15 @@ router
             return res.status(401).json({ error: 'token missing or invalid' });
         }
 
-        const ownerId = req.user.id;
-        const q = await db.query('SELECT * FROM project WHERE owner_id = $1', [
-            ownerId,
-        ]);
+        const userId = req.user.id;
+        const q = await db.query(
+            `SELECT * FROM project
+            WHERE id IN (
```

Comment:
`WHERE EXISTS (SELECT 1 FROM users__project WHERE users_id = $1 AND project_id = $2)` is more efficient... although the database might actually manage to optimize the query into that :)